harlyon/Series
# Implement react hooks and context API

#### Description?
Refactors the code to use react hooks and context API. Also, this PR updates the dependencies to the latest version.

#### How Has This Been Tested?
- git clone this repository
- git checkout to this branch `implement-react-hooks`
- Run `npm install` to install the latest dependencies
- Run `npm start` to run the app
- No major change on functionality but is working as expected
#### Any background context you want to provide?
Hooks are a new addition in React 16.8. They let you use state and other React features without writing a class

#### Screenshots (if appropriate):
N/A
